roles/sap_ha_install_pacemaker/defaults/main.yml

```diff
@@ -1,73 +1,34 @@
 ---
-
-# TYPE - 'az'
-sap_ha_install_pacemaker_type: ""
-
-# Platform - baremetal, azure, aws, gcp, rhevm, alicloud
-sap_ha_install_pacemaker_platforme: "{{ platform }}" 
-# Specific variables per node - 'primary' or 'secondary'
-sap_ha_install_pacemaker_role: ""
-
-# Part - '1' or '2'
-sap_ha_install_pacemaker_part: ""
-
 sap_ha_install_pacemaker_sid: "{{ sap_hana_sid }}"
 sap_ha_install_pacemaker_instance_number: "{{ sap_hana_instance_number }}"
 
-sap_ha_install_pacemaker_cluster_name: "nw1-azr"
+sap_ha_install_pacemaker_cluster_name: "{{ sap_hana_cluster_name }}"
+sap_ha_install_pacemaker_cluster_nodes: "{{ sap_hana_cluster_nodes }}"
 
-sap_ha_install_pacemaker_fqdn: ""
+sap_ha_install_pacemaker_vip1: " {{ sap_hana_vip1 }}}"
+sap_ha_install_pacemaker_vip2: " {{ sap_hana_vip2 }}}"
 
-sap_ha_install_pacemaker_node1_ip: ""
-sap_ha_install_pacemaker_node1_hostname: "{{ hana_node1_hostname }}"
+sap_ha_install_pacemaker_hacluster_password: "{{ sap_hana_hacluster_password }}"
 
-sap_ha_install_pacemaker_node2_ip: ""
-sap_ha_install_pacemaker_node2_hostname: "{{ hana_node2_hostname }}"
+sap_ha_install_pacemaker_stonith_devices: "{{ sap_pacemaker_stonith_devices }}"
 
-sap_ha_install_pacemaker_hacluster_password: ""
+## AZ specific definitions
+
+# TYPE - 'az'
```

Comment:
For readability of end users, can this line please list all types? Of refer back to a specific README link anchor for where these are listed?

I see the comment `## AZ specific definitions` 2 lines before, but this variable appears to be generic - should this line move upwards?

Reply:
To be honest this is not used and appear inherited from another role, there are no tasks which work with these platform specific variables. We kept the defaults as reference for future extensions.

Reply:
OK for Infrastructure Platform specific cluster configurations to be removed temporarily, however can we please raise this as a GH Issue for tracking the discussion for these tasks?

Reply:
Tracked under GH Issue #164